jetty-http/src/main/java/org/eclipse/jetty/http/pathmap/ServletPathSpec.java

```diff
@@ -26,17 +26,16 @@
 {
     private static final Logger LOG = Log.getLogger(ServletPathSpec.class);
 
-    private final String _declaration;
     private final PathSpecGroup _group;
     private final int _pathDepth;
     private final int _specLength;
     private final String _prefix;
     private final String _suffix;
 
     /**
```

Comment:
This class should have javadoc that describes what a ServletPathSpec is:  eg prefixed with "servlet|" etc etc